numpy/lib/mixins.py

```diff
@@ -68,7 +68,7 @@ class NDArrayOperatorsMixin:
     It is useful for writing classes that do not inherit from `numpy.ndarray`,
     but that should support arithmetic and numpy universal functions like
     arrays as described in `A Mechanism for Overriding Ufuncs
-    <../../neps/nep-0013-ufunc-overrides.html>`_.
+    <https://numpy.org/neps/nep-0013-ufunc-overrides.html>`_.
```

Comment:
Same comment as above - let's try intershpinx here

Reply:
Deferring NEP intersphinx enhancement for another PR.  This PR is for broken links.